lib/macho/macho_file.rb

```diff
@@ -556,7 +556,7 @@ def low_fileoff
 
       segments.each do |seg|
         sections(seg).each do |sect|
-          next if sect.size == 0
+          next if sect.size.zero?
```

Comment:
RuboCop actually pointed out both as alternatives - I just went with the first, although `sect.empty?` makes a lot more sense :+1: